Assets/MixedRealityToolkit.SDK/Inspectors/UX/Interactable/PressableButtonInspector.cs

```diff
@@ -125,87 +112,74 @@ private ButtonInfo GatherCurrentInfo()
-            newInfo.PressDistPos = DrawPlaneAndHandle(pressDistancePlane, info.TouchCageLocalBounds.size * 0.35f, newInfo.PressDistPos, info.TouchStartOrigin, "Press event", editingEnabled);
-
-            if (editingEnabled)
+            float newPressDistance = DrawPlaneAndHandle(pressPlaneVertices, info.TouchCageLocalBounds.size * 0.35f, info.PressDistance, info, "Press Distance", editingEnabled);
+            if (editingEnabled && newPressDistance != info.PressDistance)
             {
-                // Clamp the z values to target start / end
-                newInfo.PressDistPos = Mathf.Max(newInfo.StartPos, newInfo.PressDistPos);
-                newInfo.PressDistPos = Mathf.Min(newInfo.EndPos, newInfo.PressDistPos);
-                // Set based on distance from start
-                // Adjust for scaled objects
-                newInfo.PressDistance = Mathf.Abs(newInfo.PressDistPos - newInfo.StartPos) * transform.lossyScale.z;
+                EnforceDistanceOrdering(ref info);
+                info.PressDistance = Mathf.Clamp(newPressDistance, info.ReleaseDistance, info.MaxPushDistance);
             }
 
-            // RELEASE DISTANCE DELTA
-            Handles.color = Color.red;
-            newInfo.ReleaseDistPos = DrawPlaneAndHandle(releasePlane, info.TouchCageLocalBounds.size * 0.3f, newInfo.ReleaseDistPos, info.TouchStartOrigin, "Release event", editingEnabled);
-
-            if (editingEnabled)
+            // MAX PUSH
+            Handles.color = Color.cyan;
+            float newMaxPushDistance = DrawPlaneAndHandle(endPlaneVertices, info.TouchCageLocalBounds.size * 0.5f, info.MaxPushDistance, info, "Max Push Distance", editingEnabled);
+            if (editingEnabled && newMaxPushDistance != info.MaxPushDistance)
             {
-                // Clamp the z values to press distance
-                newInfo.ReleaseDistPos = Mathf.Min(newInfo.PressDistPos, newInfo.ReleaseDistPos);
-                // Set based on distance from press distance
-                // Adjust for scaled objects
-                newInfo.ReleaseDistanceDelta = (newInfo.PressDistPos - newInfo.ReleaseDistPos) * transform.lossyScale.z;
+                EnforceDistanceOrdering(ref info);
+                info.MaxPushDistance = Mathf.Max(newMaxPushDistance, info.PressDistance);
             }
 
+            /// {dahof} NOTE: collider box editing is disabled as we would need to find the min / max point of it along the press direction, which is arbitrary.
+            /// I think for simplicity reasons, we should make editing the collider independent from editing the PressableButton distances.
+
             // BUTTON CONTENT ORIGIN
             // Don't allow editing of button position
-            Handles.color = Color.green;
-            DrawPlaneAndHandle(pressStartPlane, info.TouchCageLocalBounds.size * 0.4f, newInfo.StartPos, info.TouchStartOrigin, "Moving button visuals", false);
-
+            /*Handles.color = Color.green;
```

Comment:
Should this commented out code just be removed?

Reply:
that was part of the ported code. you're right we can remove that - we can always look it up in git history